src/commonMain/resources/db/test/V3.1__TestData.sql

```diff
@@ -1,3 +1,25 @@
 INSERT INTO uploader (name, email, password, active, testplay, admin, "uniqueName") VALUES
-    ('test', 'test@test.com', '$2a$12$3zoAVlxDlvGtoVywB04Amep3qTHmgaXivA3CMHb0YdZkLEHZxFNzi', TRUE, FALSE, FALSE, 'test'),
-    ('admin', 'admin@test.com', '$2a$12$HxGPVg8oEr.igslnwc0RIONm732SVbXKg6Yu6C6kHOtKbbsEmjPCO', TRUE, TRUE, TRUE, 'admin');
\ No newline at end of file
+    ('test', 'test@test.com', '$2a$12$3zoAVlxDlvGtoVywB04Amep3qTHmgaXivA3CMHb0YdZkLEHZxFNzi', TRUE, FALSE, FALSE, 'test'),  -- password: test
+    ('admin', 'admin@test.com', '$2a$12$HxGPVg8oEr.igslnwc0RIONm732SVbXKg6Yu6C6kHOtKbbsEmjPCO', TRUE, TRUE, TRUE, 'admin'); -- password: admin
+
+INSERT INTO beatmap (name, description, uploader, bpm, duration, "songName", "songSubName", "songAuthorName", "levelAuthorName", uploaded, automapper, ranked, qualified, score, upvote, downvote) VALUES
```

Comment:
This should be in a new migration. You can't edit migrations once they've been run:
```
Exception in thread "main" org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration checksum mismatch for migration version 3.1
-> Applied to database : -604125379
-> Resolved locally    : 1406895797. Either revert the changes to the migration, or run repair to update the schema history.
```